src/otx/algo/modules/activation.py

```diff
@@ -57,7 +57,6 @@ def forward(self, x: Tensor) -> Tensor:
     nn.Sigmoid,
     Swish,
     nn.GELU,
-    nn.SiLU,
```

Comment:
we use nn.SiLU for RTMDet, could you check if it is OK to delete it? I think we should keep SiLU

Reply:
nn.SiLU has `inplace` on torch 2.4.0 that is included in develop environment.
![image](https://github.com/user-attachments/assets/e41fe5f0-3f4f-47c5-9354-8f5f5ace3409)